ansible/playbooks/sap-hana-cluster.yaml

```diff
@@ -7,10 +7,12 @@
     # is_primary is selected so that tasks that need to be issued one are honoured correctly
     is_primary: "{{ ansible_play_hosts[0] == inventory_hostname }}"
     primary_hostname: "{{ ansible_play_hosts[0] }}"
-    use_sbd: yes
+    use_sbd: "{{ default('yes') }}"
     # Azure fencing specific vars
+    identity_management:  # use SPI (service principal) or MSI (managed identity)
```

Comment:
This one needs to be set by end user manually in ansible section of config.yaml.  It defines which auth method that will be used.   
What can be maybe done is to set is as a mandatory value in case 'use_sbd = false/0' so the test fails immediately.
Need to figure out first how to do that :blush: 
But I can include it in this PR.

Reply:
> to be set by end user manually in ansible section of config.yaml

can you confirm that there's no need to change any code about it. It is enough to have in the config.yaml something like

```
ansible:
    ...
    create:
        ...
        - sap-hana-cluster.yaml -e identity_management=msi
```

If it is so simple could you please add a note about it in this section https://github.com/SUSE/qe-sap-deployment/blob/main/README.md#ansible-settings (or any better place you manage to find)

Reply:
I have added the information to docs/playbooks/README.md since it was already included there and pointed user to this document in the README you mentioned.